docs/releases/v2018.1.rst

```diff
@@ -0,0 +1,374 @@
+Gluon 2018.1 (in development)
```

Comment:
not in development anymore, right?

Reply:
this will be changed with the final commit before the releases, which also changes all version numbers across the codebase.